net/basicstation/Makefile

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (C) 2022 TDT AG <development@tdt.de>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=basicstation
+PKG_VERSION:=2.0.6
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/lorabasics/basicstation/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=7e285de94bae1174b4c3496fc3ab15fe67c72f92c0693d2320bafc654a9dfb43
+
+PKG_MAINTAINER:= Marcus Schref <mschref@tdt.de>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DEPENDS:=sx1302_hal mbedtls
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/basicstation
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=LoRaWAN
+  TITLE:=LoRa Basic Station
+  DEPENDS:=+kmod-usb-acm
+endef
+
+define Package/basicstation/description
+  LoRa Basic Station. The LoRaWAN Gateway Software.
+endef
+
+define Package/basicstation/conffiles
+/etc/config/basicstation
+endef
+
+define Build/Prepare
+	$(call Build/Prepare/Default)
+	rm -rf $(PKG_BUILD_DIR)/deps/
+	rm -f $(PKG_BUILD_DIR)/makefile
+	rm -f $(PKG_BUILD_DIR)/makefile.s2core
+	rm -f $(PKG_BUILD_DIR)/setup.gmk
```

Comment:
Default prepare does not work for this package?

Reply:
By default, Basicstation would pull dependencies (mbedtls and sx1302_hal) via some custom scripts. To avoid this behavior and use OpenWrt packages, i removed the dependencies folder and the Makefiles and added CMake. This topic was also mentioned in https://github.com/xueliu/lora-feed/issues/27.